packages/upset/src/provenance/index.ts

```diff
@@ -132,19 +128,16 @@ const bookmarkIntersectionAction = createAction<UpsetConfig, [string], Events>(
   },
 );
 
-const removeBookmarkIntersectionAction = createAction<
-  UpsetConfig,
-  [string],
-  Events
->((state, intersectionId) => {
+const removeBookmarkIntersectionAction = registry.register('remove-bookmark-intersection',
+(state, intersectionId) => {
   state.bookmarkedIntersections = state.bookmarkedIntersections.filter(
-    (id) => intersectionId !== id,
+    (id: any) => intersectionId !== id,
```

Comment:
`state.bookmarkedIntersections` could probably use better types, too. We can probably also guess a better type than `any`